SampleCode/Cxx11SGXDemo/README.txt

```diff
@@ -26,7 +26,9 @@ The project demonstrates serveral C++11 features inside the Enclave:
 How to Build/Execute the C++11 sample program
 ---------------------------------------------
 1. Install Intel(R) Software Guard Extensions (Intel(R) SGX) SDK for Linux* OS
-2. Build the project with the prepared Makefile:
+2. Make sure your environment is set:
+    a. source ${sgx-sdk-install-path}/environment
```

Comment:
change 'a.' to '$'. Same comments for other files.

Reply:
done.